### PR TITLE
new colors for explore, challenge, and instructor notes

### PR DESCRIPTION
**Pull request by erikfriesen**
added `boundless_rtd` build option for `test2` as well

### CODE REVIEW

File: test2/build.xml [hunk 1]
```diff
@@ -19,6 +19,20 @@
           description="Test build of learning and slides">
   </target>
 
+  <target name="boundless_rtd" depends="init"
+          description="Test html sphinx_rtd_theme (as used by server team)">
```
Comment (jodygarnett) [inline test2/build.xml]: This description is incorrect now :) Is this "test2" still being useful (it was initial to experiment with different slide themes).

File: boundless_rtd/static/css/boundless.css [hunk 2]
```diff
@@ -178,3 +178,35 @@ a.reference.external::after
 	  padding: 2.4px 6px;
 	  margin: auto 2px;
 }
+
+/** Color schemes for admonitions **/
+/* Color scheme for explore admonition */
+/* title bar */
+div.admonition-explore > p.admonition-title
```
Comment (jodygarnett) [inline boundless_rtd/static/css/boundless.css]: While we are here can we look at the ❗️ symbol used for these boxes? We should be able to grab a more appropriate symbol to indicate "activity" or "play".

File: boundless_rtd/static/css/boundless.css [hunk 3]
```diff
@@ -178,3 +178,35 @@ a.reference.external::after
 	  padding: 2.4px 6px;
 	  margin: auto 2px;
 }
+
+/** Color schemes for admonitions **/
+/* Color scheme for explore admonition */
```
Comment (jodygarnett) [inline boundless_rtd/static/css/boundless.css]: Thanks for the clear comments, this will really help when we update base sphinx_rtd_theme.

**Comment by jodygarnett** [comment]
@erikfriesen I have approved the above (as non fatal), there are two comments for discussion.

**Comment by erikfriesen** [comment]
> Actually I should reject since you made this PR from your master branch bomb timer_clock boom

how did i miss that? :man_facepalming:

File: test2/build.xml (hunk 1 above)
Comment (erikfriesen) [reply test2/build.xml]: Gonna toss this one out. Made the edit to help test a very specific bit of weirdness on my machine. Not critical/relevant to the rest of this PR.

File: boundless_rtd/static/css/boundless.css (hunk 2 above)
Comment (erikfriesen) [reply boundless_rtd/static/css/boundless.css]: Can do. I'll gather up a list of candidates and share them with the team.

**Comment by erikfriesen** [comment]
Closing and sending PR from a branch

**Pull request closed without merging**